src/main/java/seedu/duke/parsermanager/ParseAddProperty.java

```diff
@@ -128,7 +129,7 @@ public ParseAddProperty(String addCommandDescription, PropertyList propertyList)
         unitTypeHashMap.put(UNIT_TYPE_LANDED_TERRENCE_SHORT, ACTUAL_UNIT_TYPE_LANDED_TERRENCE);
         unitTypeHashMap.put(UNIT_TYPE_SEMI_DETEACHED_SHORT, ACTUAL_UNIT_TYPE_SEMI_DETACHED);
         unitTypeHashMap.put(UNIT_TYPE_BUNGALOW_SHORT, ACTUAL_UNIT_TYPE_BUNGALOW);
-        UNIT_TYPE_HASHMAP = unitTypeHashMap;
+        unitTypeHashmap = unitTypeHashMap;
```

Comment:
This looks like the self assignment of a variable. Try not to trip up the reader